damus/Views/AttachMediaUtility.swift

```diff
@@ -170,26 +170,26 @@ enum ImageUploader: String, CaseIterable, Identifiable {
 
     var model: Model {
         switch self {
-        case .nostrBuild:
-            return .init(index: -1, tag: "nostrBuild", displayName: NSLocalizedString("NostrBuild", comment: "Dropdown option label for system default for NostrBuild image uploader."))
-        case .nostrImg:
-            return .init(index: 0, tag: "nostrImg", displayName: NSLocalizedString("NostrImg", comment: "Dropdown option label for system default for NostrImg image uploader."))
+        case .nostrbuild:
+            return .init(index: -1, tag: "nostr.build", displayName: NSLocalizedString("nostr.build", comment: "Dropdown option label for system default for nostr.build image uploader."))
```

Comment:
You can just remove the `NSLocalizedString` and use the string directly at this point. There's no reason to localize a domain name.

Reply:
Makes sense. Removed.